examples/FirebaseNeoPixel_ESP8266/FirebaseNeoPixel_ESP8266.ino

```diff
@@ -0,0 +1,65 @@
+//
```

Comment:
If you "git mv" (Like it looks like you did with the rest of the directory these shouldn't we keep the history?